### PR TITLE
Cosullivan/add hypermedia jsonapi webapi to implementations

### PR DESCRIPTION
**Pull request by cosullivan**
Hi,

I've added two new packages to the implementations, one is a .NET client implementation and the other is a .NET server implementation. 

I've also add a new link to the Examples section.

Thanks,
Cain

### CODE REVIEW

**Comment by dgeb** [comment]
@cosullivan Sorry to let this PR sit. Could you please rebase and squash it down to a single commit per implementation?

**Comment by cosullivan** [comment]
Hi @dgeb no worries about the delay. I've just rebased and squashed.

Thanks,
Cain

File: implementations/index.md [hunk 1]
```diff
@@ -159,6 +162,11 @@ has a page describing how to emit conformant JSON.
 
 * [RestPack::Serializer provides examples](http://restpack-serializer-sample.herokuapp.com/) which demonstrate sample responses.
 * [Endpoints provides a fully working example API](http://github.com/endpoints/example/)
+* [http://hypermedia.cainosullivan.com](http://hypermedia.cainosullivan.com/) is a live JSON API service that displays a subset of the data from the 
+mythology.stackexchange.com site.
+  * [http://hypermedia.cainosullivan.com/v1/users](http://hypermedia.cainosullivan.com/v1/users?$prettify=yes)
+  * [http://hypermedia.cainosullivan.com/v1/posts](http://hypermedia.cainosullivan.com/v1/posts?$prettify=yes)
+  * [http://hypermedia.cainosullivan.com/v1/comments](http://hypermedia.cainosullivan.com/v1/comments?$prettify=yes)
```
Comment (dgeb) [inline implementations/index.md]: I'd rather not add this example unless it has some docs, description of available URLs, etc. We don't have room here to include a bunch of example URLs.

**Comment by dgeb** [comment]
@cosullivan Could you please rebase and squash the 7 commits into a single one? A fresh branch and PR would also be fine.

File: implementations/index.md (hunk 1 above)
Comment (cosullivan) [reply implementations/index.md]: Ok, I have removed and pushed.

Thanks,
Cain

**Comment by cosullivan** [comment]
Ok, let me start this again :-)

**Pull request closed without merging**